test/uri.jl

```diff
@@ -477,4 +478,27 @@ end
     # Issue 323
     @test string(HTTP.URI(scheme="http", host="example.com")) == "http://example.com"
 
+    @testset "Normalize URI paths" begin
+        # Examples given in https://tools.ietf.org/html/rfc3986#section-5.2.4
+        @test URIs.normpath("/a/b/c/./../../g") == "/a/g"
+        @test URIs.normpath("mid/content=5/../6") == "mid/6"
+
+        checknp = (x, y)->(@test URIs.normpath(URI(x)) == URI(y))
```

Comment:
why not write ? 
```suggestion
        checknp(x, y) = @test URIs.normpath(URI(x)) == URI(y)
```

(For my education)

Reply:
That would probably be fine, I'm just never sure of the exact scoping/context rules for defining functions inside of `@testset`s, so I figured it's easier to just assign a closure.